administration/bundles.md

```diff
@@ -40,9 +40,26 @@ openhab> bundle:list
 
 The _start_ command works accordingly:
 
-```
+```python
 openhab> bundle:start 209
 openhab> bundle:list
 ...
 209 | Active   |  80 | 2.0.0.b3              | Network Binding
 ```
+
+## Naming Convention For Bundles
+
+Bundles are named according to the following convention.
+
+```java
+<prefix>-<type>-<id>
+```
+
+where
+
+- **prefix** is ALWAYS "openhab" 
+- **type** is the add-on type, e.g. "binding" or "ui"
+- **id** is the identifier for this bundle
+  (It is also the id that you would have to use openHAB 1.x in addons.cfg, e.g. "hue", "map", etc.)
```

Comment:
Consider, "This is the id you would have used in the openHAB 1.x addons.cfg file (e.g. "hue", "map", etc.)"

Reply:
I think the id is used for many other things internally, so putting the focus only on the 1.x usage could be misunderstood.

@ThomDietrich  wdyt?

Reply:
I know you ask @ThomDietrich , but my two cents are as follows... "OH1.x??  What's that?".  OH1 is in the rear view mirror, and getting farther away all the time.  At some point we would remove the reference anyway.  I don't see this adds value, and it potentially adds confusion for someone who is just coming on board.

Reply:
I am asking him for a third opinion. :)

After reading your 2 cents my suggestion would be to remove that sentence completely from the beginning.

Reply:
I agree to remove that additional detail all together.

Reply:
Already done in the last commit. :)